### ISSUE TITLE
Check `--doctor` on Windows

### ISSUE THREAD

**Issue by VorpalBlade** [VorpalBlade/chezmoi_modify_manager#38]
The PATH logic needs to be verified on Windows. What about the .exe file extension specifically?

**Comment by VorpalBlade** [comment]
Probably fixed by f7844ccfb429fbad21df99d0c8d4af5ca8e01ee6

**Comment by VorpalBlade** [comment]
@lukaszkadlubowski I have just released 2.1.5 that includes the likely fix for this.

I would appreciate if you could test that
1. The self updater `-u` works on Windows (if it doesn't that is good to know, but I wouldn't be able to do much except report it onwards to the crate [what libraries are called in the Rust world] that implements this)
2. Check if `--doctor` works as expected with this change.

**Comment by lukaszkadlubowski** [comment]
I confirm that it works like a charm, here it is the first part of the `--doctor` output:

```
PS C:\Users\lk\.local\share\chezmoi> chezmoi_modify_manager --doctor 
RESULT    CHECK                MESSAGE
Info      version              2.1.5
Info      rustc-version        1.72.1
Info      host                 os=windows, arch=x86_64, info=Windows 10.0.19045 (Windows 10 Pro) [64-bit]
Info      has-chezmoi          Chezmoi found. Version: chezmoi version v2.40.0, commit 6a8ca1634654734bb33a036ffb9c21e6b9f4d28d, built at 2023-09-19T09:56:08Z, built by goreleaser
Ok        in-path              chezmoi_modify_manager is in PATH at C:\Users\lk\AppData\Local\chezmoi_modify_manager\chezmoi_modify_manager.exe
Info      has-ignore           Ignore of **/*.src.ini found

Output of chezmoi doctor:
RESULT    CHECK                       MESSAGE
ok        version                     v2.40.0, commit 6a8ca1634654734bb33a036ffb9c21e6b9f4d28d, built at 2023-09-19T09:56:08Z, built by goreleaser
warning   latest-version              v2.40.2
ok        os-arch                     windows/amd64
ok        systeminfo                  Microsoft Windows 10 Pro (10.0.19045 N/A Build 19045)
ok        go-version                  go1.21.1 (gc)
ok        executable                  ~/AppData/Local/Microsoft/WinGet/Packages/twpayne.chezmoi_Microsoft.Winget.Source_8wekyb3d8bbwe/chezmoi.exe 
ok        upgrade-method              replace-executable
etc. ...
```
